### PR TITLE
Add support for copying tomviz.html from source

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

**Comment by cjh1** [comment]
@cryos I think this provides the behavior you were expecting.

File: CMakeLists.txt [hunk 1]
```diff
@@ -94,15 +94,18 @@ endif()
 # Add web application
 # -----------------------------------------------------------------------------
 option(TOMVIZ_DOWNLOAD_WEB  "Enable downloading web application." OFF)
-set(tomviz_html_path "${CMAKE_BINARY_DIR}/web/tomviz.html")
+set(tomviz_html_source_path "${CMAKE_SOURCE_DIR}/web/tomviz.html")
```
Comment (jourdain) [inline CMakeLists.txt]: I think the actual path is `${CMAKE_BINARY_DIR}/web/www/tomviz.html`

Comment (cjh1) [inline CMakeLists.txt]: Does that matter isn't it the install location that is important? I am happy to change it but would like to understand why?

Comment (jourdain) [inline CMakeLists.txt]: The path I gave you, is the one that is used when building/generating that file. 
I can probably change it to be the root directory of the web, but conceptually, I'd rather keep it separate.

Comment (cjh1) [inline CMakeLists.txt]: ok, will update

**Pull request merged**